modelforge/potential/processing.py

```diff
@@ -449,10 +450,10 @@ def __init__(self, strategy: str, cutoff: unit.Quantity):
             The cutoff function applied to the pairwise distances.
         """
         super().__init__()
-        from .utils import CosineAttenuationFunction
+        from .utils import PhysNetAttenuationFunction
 
         self.strategy = strategy
-        self.cutoff_function = CosineAttenuationFunction(cutoff)
+        self.cutoff_function = PhysNetAttenuationFunction(cutoff)
```

Comment:
Does it make sense to have the cutoff function be something the user could set (if the idea is to be able to eventually use this function in a more modular context)? The cosine function should still avoid a singularity at r_ij=0 (and the functions are still very closely related).   I'm not saying we necessarily have to make this user modifiable now.

Reply:
Discussion offline: This might be something we want to be able to use this in a more general sense (although right now, the goal is to be able to implement physnet).